### PR TITLE
Complete template scenarios

### PR DESCRIPTION
**Pull request by jyotheeshwar**
@ansonliang55 review please. 

Thanks!

### CODE REVIEW

File: template.feature [hunk 1]
```diff
@@ -121,31 +123,75 @@ I should see the image <context> "<value>" is enabled|disabled
...
+
+increase/decrease
+I <option> the "<slider_name>" slider <context> "<value>"'s value to "<value>"
+
+Examples:
+	| context    | option   |
+	| in section | increase |
+	| with id    | decrease |
+	eg. I see the "rating" slider with id "rating-slider" has value "4"
+	eg. I increase the "rating" slider with id "rating-slider"'s value to "10"
```
Comment (ansonliang55) [inline template.feature]: I am not sure 's is a good way to templating. I prefer it saying, I increase the value of "rafting" slider ...

Also, increase and decrease slider doesn't make sense to me. I would prefer sliding the slider or setting the slider. What you think?

File: template.feature [hunk 2]
```diff
@@ -121,31 +123,75 @@ I should see the image <context> "<value>" is enabled|disabled
...
+	| not see    |
+
+Lists:
+Visibility: 	
+I should <visibility> a list item labelled "<string>"	
+Examples: 
+	| visibility |
+	| see        |
+	| not see    |
+	eg. I should see a list item labelled "Health levels"
```
Comment (ansonliang55) [inline template.feature]: I should see a "Health levels" list item?

**Comment by ansonliang55** [comment]
+1 otherwise

File: template.feature (hunk 1 above)
Comment (jyotheeshwar) [inline template.feature]: yeah I agree. the sentence formation seems better the way you've mentioned it. 

Increase decrease would map to the values right? You slide the slider to the right or left, to increase or decrease. But that wouldn't cover vertical sliders.

File: template.feature (hunk 2 above)
Comment (jyotheeshwar) [inline template.feature]: yup. will update it.

File: template.feature (hunk 1 above)
Comment (ansonliang55) [inline template.feature]: ok use increaes/decrease then

**Comment by jyotheeshwar** [comment]
merged to master.

**Pull request closed without merging**